spec/mailers/user_mailer_spec.rb

```diff
@@ -659,11 +661,158 @@ def expect_email_body_to_have_help_and_contact_links
+      context 'For In-Person Proofing (IPP)' do
+        context 'template displays modified content' do
+          it 'conditionally renders content in the what to expect section applicable to IPP' do
+            aggregate_failures do
+              [
+                t('in_person_proofing.headings.barcode'),
+                t('in_person_proofing.process.state_id.heading'),
+                t('in_person_proofing.process.state_id.info'),
+              ].each do |copy|
+                Array(copy).each do |part|
+                  expect(mail.html_part.body).to have_content(part)
+                end
+              end
+            end
+          end
+        end
+
+        it 'renders Questions? and Learn more link only once' do
+          expect(mail.html_part.body).to have_content(
+            t('in_person_proofing.body.barcode.questions'),
+          ).once
+          expect(mail.html_part.body).to have_link(
+            t('in_person_proofing.body.barcode.learn_more'),
+            href: MarketingSite.help_center_article_url(
+              category: 'verify-your-identity',
+              article: 'verify-your-identity-in-person',
+            ),
+          ).once
+        end
+
+        it 'template does not displays Enhanced In-Person Proofing specific content' do
+          aggregate_failures do
+            [
+              t('in_person_proofing.headings.barcode_eipp'),
+              t('in_person_proofing.process.state_id.heading_eipp'),
+              t('in_person_proofing.process.state_id.info_eipp'),
+              t('in_person_proofing.headings.barcode_what_to_bring'),
+              t('in_person_proofing.body.barcode.what_to_bring'),
+              t('in_person_proofing.process.eipp_bring_id.heading'),
+              t('in_person_proofing.process.eipp_bring_id.info'),
+              t('in_person_proofing.process.eipp_what_to_bring.heading'),
+              t('in_person_proofing.process.eipp_what_to_bring.info'),
+              t('in_person_proofing.process.eipp_state_id_passport.heading'),
+              t('in_person_proofing.process.eipp_state_id_passport.info'),
+              t('in_person_proofing.process.eipp_state_id_military_id.heading'),
+              t('in_person_proofing.process.eipp_state_id_military_id.info'),
+              t('in_person_proofing.process.eipp_state_id_supporting_docs.heading'),
+              t('in_person_proofing.process.eipp_state_id_supporting_docs.info'),
+            ].each do |copy|
+              Array(copy).each do |part|
+                expect(mail.html_part.body).to_not have_content(part)
+              end
+            end
+          end
+
+          t('in_person_proofing.process.eipp_state_id_supporting_docs.info_list').each do |item|
+            expect(mail.html_part.body).to_not have_content(strip_tags(item))
+          end
+        end
+      end
+
+      context 'For Enhanced In-Person Proofing (Enhanced IPP)' do
+        is_enhanced_ipp = true
```

Comment:
Nit: I would expect to use a let block here.  If there's a reason that a let block wouldn't work, let me know.

Reply:
Fixed with commit 01b0b3c2481783972b0dbb00ac08ab8f89dc9546